go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/containerd/cgroups/v3 v3.0.3
 	github.com/containerd/console v1.0.3
 	github.com/containerd/continuity v0.4.3
+	github.com/containerd/errdefs v0.1.0
```

Comment:
WDYT
- https://github.com/containerd/errdefs/issues/4

Reply:
We should, I purposely tagged 0.1.0 as the same version from containerd. I think we can split in 0.2.0 and include the additional errors.